packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -27,3 +27,13 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module)
+  .addDecorator(ampDecorator)
+  .add('AMP Audio skin', () => (
+    <AmpMediaPlayer
```

Comment:
I'm getting a console warning: `<iframe gesture="media"> is not supported. Use <iframe allow="autoplay">,`

Reply:
This console warning would've been present previously, though the attribute can be removed as it is not supported. It was incorrectly referenced in old Google docs for Chrome.

https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#iframe

Reply:
This console warning comes from SMP, they are planning to remove it in the future. It can be see on any instance of SMP - https://www.bbc.co.uk/sounds/play/live:bbc_radio_one

![image](https://user-images.githubusercontent.com/7791726/66043503-394f0800-e517-11e9-8f9d-c2c6e13de1f4.png)